src/exampleTypes/ResultTable.js

```diff
@@ -53,8 +53,9 @@ let HighlightedColumnHeader = observer(
       _.compact,
       _.isEmpty
     )(results),
+    showOtherMatches,
   }) =>
-    hasAdditionalFields ? (
+    hasAdditionalFields && showOtherMatches ? (
```

Comment:
this should be a property on the `node`, because if it's false we should reduce the highlighting fields to just `include` in contexture-elasticsearch